scss/tech.scss

```diff
@@ -28,4 +28,19 @@
     grid-area: feature-card;
     justify-content: center;
   }
+
+  @media screen and (max-width: 1024px) {
```

Comment:
this media query doesn't have a class associated, where are you applying this styles?

Reply:
yes, it does, the associated class is .tech